salt/states/zabbix_host.py

```diff
@@ -397,12 +415,19 @@ def _interface_format(interfaces_data):
                 # combine connection_args, inventory, and clear_old
                 sum_kwargs = dict(new_inventory)
```

Comment:
Is already a `dict`.

Reply:
Good catch. I'll change it.

Reply:
Done

Reply:
Ooops, the `dict` in `dict(new_inventory)` is there to copy `new_inventory` to `sum_kwargs`. Changed to `new_inventory.copy()` so this line will not induce other people to this same error and remove it again.